examples/templates/aws-ecs-container/main.tf

```diff
@@ -102,9 +102,10 @@ resource "coder_agent" "coder" {
   startup_script         = <<-EOT
     set -e
 
-    # install and start code-server
-    curl -fsSL https://code-server.dev/install.sh | sh -s -- --version 4.8.3
-    code-server --auth none --port 13337 >/tmp/code-server.log 2>&1 &
```

Comment:
Did we want to remove logging from this template (same for the others)?

Reply:
Yet another oversight on my part 😅